homeworks/andrii.tkachenko_Hodr1337/5-arrays/arrays.js

```diff
@@ -0,0 +1,31 @@
+function elka(height) {
+    let x = 0;
+    let y = 0;
+    let space; let asterics;
```

Comment:
Please, init all variables inside the loop because you don't use them outside of the loop.
Also, don't init two variables in a row. Make it on separate lines